src/il_representations/scripts/hp_tuning.py

```diff
@@ -0,0 +1,299 @@
+from il_representations.algos import augmenters, pair_constructors, encoders, losses, batch_extenders
```

Comment:
Same comment on this file as on `chain_configs.py`: it seems to be a duplicate of another file in the `configs/` directory. I assume this one is old and we should remove it?

Reply:
Yes! These two files are mistakenly kept under `scripts`. I have already removed it.